modules/custom/dkan_alt_api/dkan_alt_api.routing.yml

```diff
@@ -0,0 +1,39 @@
+dkan_alt_api.1.metastore.schemas.id.items:
+  path: '/alt/api/1/metastore/schemas/{schema_id}/items'
+  methods: [GET]
+  defaults:
+    { _controller: '\Drupal\dkan_metastore\WebServiceApi::getAll'}
+  requirements:
+    _permission: 'metastore increased visibility'
+  options:
+    _auth: ['basic_auth']
+
+dkan_alt_api.1.metastore.schemas.id.items.id:
+  path: '/alt/api/1/metastore/schemas/{schema_id}/items/{identifier}'
+  methods: [GET]
+  defaults:
+    { _controller: '\Drupal\dkan_metastore\WebServiceApi::get'}
+  requirements:
+    _permission: 'metastore increased visibility'
```

Comment:
@thierrydallacroce I do not believe this permission exists.

Reply:
@fmizzell good catch, will fix it shortly. Thanks.